content/en/docs/pipelines/installation/overview.md

```diff
@@ -8,15 +8,41 @@ Kubeflow Pipelines offers a few installation options.
 This page describes the options and the features available
 with each option:
 
-* A [standalone](#standalone) Kubeflow Pipelines deployment.
-* Kubeflow Pipelines as [part of a full Kubeflow deployment](#full-kubeflow).
-* **Alpha**: [GCP Hosted ML Pipelines](#marketplace).
+* A [standalone](#standalone) Kubeflow Pipelines deployment is the minimal
+portable installation that only includes Kubeflow Pipelines.
+* Kubeflow Pipelines as [part of a full Kubeflow deployment](#full-kubeflow) provides
+all Kubeflow components and more integration with each platform.
+* **Beta**: [AI Platform Pipelines](#marketplace) provides easy-to-use management
+UI for installing and using Kubeflow Pipelines on GCP.
```

Comment:
to nitpick: a 'standalone' installation also does show up in the CAIP UI, and I think people would find that info useful/convenient.